tiers/checks.go

```diff
@@ -262,11 +287,24 @@ func numberOfLimitRanges(number int) innerObjectCheck {
 	}
 }
 
-func numberOfNetworkPolicies(number int) innerObjectCheck {
+func numberOfNetworkPolicies(number int) namespaceObjectsCheck {
 	return func(t *testing.T, ns *v1.Namespace, memberAwait *wait.MemberAwaitility, userName string) {
 		nps := &netv1.NetworkPolicyList{}
 		err := memberAwait.Client.List(context.TODO(), nps, providerMatchingLabels, client.InNamespace(ns.Name))
 		require.NoError(t, err)
 		assert.Len(t, nps.Items, number)
 	}
 }
+
+func numberOfClusterResourceQuotas(number int) clusterObjectsCheck {
+	return func(t *testing.T, memberAwait *wait.MemberAwaitility, userName string) {
+		quotas := &quotav1.ClusterResourceQuotaList{}
+		matchingLabels := client.MatchingLabels(map[string]string{ // make sure we only list the ClusterResourceQuota resources associated with the given "userName"
+			"toolchain.dev.openshift.com/provider": "codeready-toolchain",
+			"toolchain.dev.openshift.com/owner":    userName,
```

Comment:
could we also verify the other labels - `toolchain.dev.openshift.com/type`,`toolchain.dev.openshift.com/tier` and `toolchain.dev.openshift.com/revision`?